lib/init.js

```diff
@@ -14,7 +14,7 @@ function replaceSchemasAndTheme(theme, input, output, schemas) {
   replace({
     regex: '###schemas###',
     replacement: schemas
-      .map(function (schema) { return `  require('${path.join(prefix, schema)}')`; })
+      .map(function (schema) { return `  require('${path.join(prefix, schema).replace('\\', '\\\\')}')`; })
```

Comment:
Not sure why this replace is needed but if it's necessary, probably should use/check `path.sep` at least.